src/components/graphs/DataByHourGraph/index.tsx

```diff
@@ -346,6 +373,27 @@ function DataByHourGraph({ id, stats = [] }: Props) {
                     position: 'top',
                     silent: true,
                 },
+                {
+                    data: [renderingTimezone],
+                    axisLabel: {
+                        align: 'center',
+                    },
+                    axisLine: {
+                        show: false,
+                    },
+                    axisPointer: {
+                        show: false,
+                    },
+                    axisTick: {
+                        show: false,
+                    },
+                    tooltip: {
+                        show: false,
+                    },
+                    offset: 20,
+                    position: 'bottom',
+                    silent: true,
```

Comment:
echarts typing means it complained while trying to share these settings that are nearly identical to the one right up above :(